test/surface_albedo.jl

```diff
@@ -0,0 +1,217 @@
+        simulation = ClimaAtmos.get_simulation(config)
+        (; u, p, t) = simulation.integrator
+        ClimaAtmos.set_surface_albedo!(u, p, t, p.atmos.surface_albedo)
+        @test all(
+            isapprox.(
+                p.radiation.radiation_model.direct_sw_surface_albedo,
+                0.38;
+                atol = eps(FT),
+            ),
+        )
+        @test all(
+            isapprox.(
+                p.radiation.radiation_model.diffuse_sw_surface_albedo,
+                0.38;
+                atol = eps(FT),
+            ),
+        )
+
+        # test set_surface_albedo!(Y, p, t, α_type::RegressionFunctionAlbedo)
+        config.parsed_args["rad"] = "clearsky"
+        config.parsed_args["albedo_model"] = "RegressionFunctionAlbedo"
+        simulation = ClimaAtmos.get_simulation(config)
+        (; u, p, t) = simulation.integrator
+
+        ClimaAtmos.set_surface_albedo!(u, p, t, p.atmos.surface_albedo)
+
+        # test that the albedo is initiated (not NaN) - values are checked below
+        @test !isnan(sum(p.radiation.radiation_model.direct_sw_surface_albedo))
+        @test !isnan(sum(p.radiation.radiation_model.diffuse_sw_surface_albedo))
+    end
+end
+
+@testset "values of RegressionFunctionAlbedo against Jin et al 2011" begin
+    FT = Float32
+
+    # DIRECT
+    # these values were visually confirmed by comparing `α_dir_list` generated using this script with Figure 2 from Jin et al 2011 for n = 1.34
+    correct_vals = [
+        [
+            0.021,
+            0.021,
+            0.021,
+            0.022,
+            0.024,
+            0.03,
+            0.044,
+            0.078,
+            0.165,
+            0.379,
+            0.753,
+        ],
+        [
+            0.021,
+            0.021,
+            0.021,
+            0.022,
+            0.024,
+            0.03,
+            0.045,
+            0.078,
+            0.154,
+            0.313,
+            0.515,
+        ],
+        [
+            0.022,
+            0.022,
+            0.022,
+            0.023,
+            0.025,
+            0.032,
+            0.046,
+            0.077,
+            0.135,
+            0.231,
+            0.321,
+        ],
+        [
+            0.008,
+            0.008,
+            0.008,
+            0.009,
+            0.01,
+            0.013,
+            0.022,
+            0.045,
+            0.114,
+            0.316,
+            0.753,
+        ],
+        [
+            0.008,
+            0.008,
+            0.008,
+            0.009,
+            0.01,
+            0.013,
+            0.022,
+            0.045,
+            0.107,
+            0.261,
+            0.515,
+        ],
+        [
+            0.008,
+            0.008,
+            0.009,
+            0.009,
+            0.01,
+            0.014,
+            0.023,
+            0.044,
+            0.093,
+            0.193,
+            0.321,
+        ],
+    ]
+
+    n_list = FT.([1.34, 1.2])
```

Comment:
If the values correspond to n=1.34, should we only test n=1.34 here?

Reply:
This is testing the values used in the paper.

Reply:
Oh, I see! I was misled by the above comments "# these values were visually confirmed by comparing `α_dir_list` generated using this script with Figure 2 from Jin et al 2011 for n = 1.34". Should it be for "n=1.34 and n=1.20"?